dtfs-central-api/src/helpers/get-fee-record-payment-entity-group-status.ts

```diff
@@ -0,0 +1,29 @@
+import { FEE_RECORD_STATUS, FeeRecordStatus } from '@ukef/dtfs2-common';
+import { FeeRecordPaymentEntityGroup } from '../types/fee-record-payment-entity-group';
+
+/**
+ * Gets the status of the fee record payment entity group
+ * @param group - The fee record payment entity group
+ * @returns The status of the group
+ * @throws {Error} If the group has an empty fee records array
+ * @throws {Error} If the group fee record statuses are invalid
+ */
+export const getFeeRecordPaymentEntityGroupStatus = (group: FeeRecordPaymentEntityGroup): FeeRecordStatus => {
+  if (group.feeRecords.length === 0) {
```

Comment:
```suggestion
  if (!group?.feeRecords?.length) {
```

Reply:
The explicit check for length not being zero better communicates the intention of what we're checking. Similarly, the optional chaining isn't required at this stage of the code because of typescript

Reply:
I am not too concerned on above as we seems to have both ` === 0` and `!` both have advantages and disadvantages, I am happy to resolve on this occasion given it is not a well established standard.